content/blog/2025-05-13-brilooped/index.md

```diff
@@ -0,0 +1,443 @@
++++
+title = "Brilooped"
+[[extra.authors]]
+name = "Gerardo Garcia Teruel Noriega"
+[[extra.authors]]
+name = "Dev Patel"
++++
+
+# Brilooped
+
+## Abstract
+
+In this project we introduce Briloop, an extension into the [Bril programming language](https://capra.cs.cornell.edu/bril/) that supports structured control flow, this allows bril programs to use if-then-else, while, and block statements instead of relying on jumps and branches. We also implemented Ramsey's [Beyond Relooper](https://dl.acm.org/doi/10.1145/3547621) to translate bril programs into briloop automatically, having a an average performance penalty of 9% when using the core bril benchmarks. Brilooped not only enables programmers to write bril programs with structured control flow, but also enables a translation from Bril into WebAssembly.
```

Comment:
Let's try to keep capitalization consistent throughout the post: bril -> Bril and briloop -> Briloop.